test/unit/test-amp-story-player.js

```diff
@@ -385,6 +389,90 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
       );
     });
 
+    it('adds stories programmatically', async () => {
+      buildStoryPlayer(3);
+      await manager.loadPlayers();
+
+      const storyObjects = createStoryObjects(1);
+      playerEl.add(storyObjects);
+
+      const stories = toArray(playerEl.querySelectorAll('a'));
+
+      expect(stories.length).to.eql(4);
+    });
+
+    it.only('throws an error if the add method is not passed an array', async () => {
```

Comment:
Here and below, please remove the `.only`

Reply:
Done in latest commits